views/js/previewer/adapter/item/qtiItem.js

```diff
@@ -74,22 +64,48 @@ define([
          * @param {String} uri - The URI of the item to load
          * @param {Object} state - The state of the item
          * @param {Object} [config] - Some config entries
-         * @param {String} [config.serviceCallId='previewer'] - The service call Id to send to the server
+         * @param {Object[]} [config.plugins] - Additional plugins to load
          * @param {String} [config.fullPage] - Force the previewer to occupy the full window.
          * @param {String} [config.readOnly] - Do not allow to modify the previewed item.
          * @returns {Object}
          */
-        init: function init(uri, state, config) {
-            config = _.defaults(config || {}, {
-                provider: 'qtiItemPreviewer',
-                serviceCallId: 'previewer'
-            });
+        init(uri, state, config = {}) {
+
+            const plugins = Array.isArray(config.plugins) ? [...defaultPlugins, ...config.plugins] : defaultPlugins;
+            const testRunnerConfig = {
+                testDefinition: 'test-container',
+                serviceCallId: 'previewer',
+                providers: {
+                    runner: {
+                        id: 'qtiItemPreviewer',
+                        module: 'taoQtiTestPreviewer/previewer/provider/item/item',
+                        bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
+                        category: 'previewer'
+                    },
+                    "proxy": {
```

Comment:
What is the reason behind mixing quoted key names and unquoted key names? I think you should not quote them.